forgero-extended/src/main/resources/data/forgero/packs/axe-additions/schematic/tree_feller_head.json

```diff
@@ -89,7 +89,7 @@
       {
         "type": "BLOCK_BREAKING_PATTERN",
         "tags": [
-          "minecraft:logs"
+          "forgero:vein_mining_logs"
         ],
         "pattern": [
```

Comment:
I think we need to add another vein mining algorithm that goes straight up or down in a line to replicate this functionality, but done properly. Again, this is one me, so I'll probably do it at some point